Tests/MessageGeneratorTests.swift

```diff
@@ -1,5 +1,255 @@
+        let coordinate = CLLocationCoordinate2D(latitude: 50.73390258, longitude: 7.09812965)
+        self.location = Location(coordinate: coordinate, city: "Bonn", country: "Germany")
+    }
+
+    override func tearDown() {
+        super.tearDown()
+    }
+
+    func testSolstice() {
+        let expected = [
+            "Today is the winter solstice ☀️! From now on, every day should get a bit brighter! Enjoy!",
+            "Sadly, today is a tiny bit shorter than yesterday. Enjoy it while it lasts!",
+            "We’ve reached the tipping point: we’ll have more sunlight every day now!",
+            "We’ve reached the tipping point: we’ll have more sunlight every day now!",
+            "We’ll have about a minute of extra light today. It’s upwards from here.",
+            "We’ve reached the tipping point: we’ll have more sunlight every day now!",
+            "About a minute of extra light. You’ll start noticing the difference soon!",
+            "About a minute of extra light. You’ll start noticing the difference soon!",
+            "Little less than a minute of extra sunlight today. It’s getting better!",
+            "We’ll have about a minute of extra light today. It’s upwards from here.",
+            "We’ll have about a minute of extra light today. It’s upwards from here.",
+            "Bring out your shorts, because today has 1 minute more sunlight.",
+            "Smile! Today has 1 minute more daylight than yesterday!",
+            "After darkness comes daylight. 1 minute more to be precise!",
+            "Today is 1 minute longer. It’s getting better and better!",
+            "1 minute more daylight today. Just let it sink in…",
+            "2 minutes extra sunshine today. Make them count!",
+            "After darkness comes daylight. 2 minutes more to be precise!",
+            "Today is 2 minutes longer than yesterday. Happy days!",
+            "Make sure to soak up that vitamin D. 2 minutes more daylight today!",
+            "2 minutes more daylight today. Just let it sink in…",
+            "Today is 2 minutes longer. It’s getting better and better!",
+            "2 minutes more daylight today. Just let it sink in…",
+            "2 minutes extra sunshine today. Make them count!",
+            "Make sure to soak up that vitamin D. 2 minutes more daylight today!",
+            "2 minutes extra sunshine today. Make them count!",
+            "2 minutes more daylight today. Just let it sink in…",
+            "Today is 2 minutes longer than yesterday. Happy days!",
+            "Bring out your shorts, because today has 2 minutes more sunlight.",
+            "Smile! Today has 3 minutes more daylight than yesterday!",
+            "Smile! Today has 3 minutes more daylight than yesterday!",
+        ]
+
+        let initialDate = self.initialDate
+        let futureDates = initialDate.futureDates()
```

Comment:
not added in this PR but I had to look up the implementation of this to understand why you knew how many dates this would produce. Worth renaming to `next30Days()`?

Reply:
Yeah, a lot of the methods and properties on this project would be worth renaming. Where's the Refactoring tool for Swift? "4.0"…… 😓

Reply:
Uhhh...in Xcode 9? Has been working well for me.

Reply:
Huh, guess it does work now! 👏🏽